boards/arm/gd32f450z_eval/support/openocd.cfg

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2022, Rtone.
```

Comment:
Don`t forget to keep rights if you copy and adapted this part.

```diff
+# Copyright (c) 2021, ATL-Electronics
+# Copyright (c) 2021, Teslabs Engineering S.L.
# Copyright (c) 2022, Rtone.
```

Reply:
oupsy, got it, thx!